crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -357,8 +360,10 @@ impl<'a> VM<'a> {
 
     // JUMPDEST operation
     pub fn op_jumpdest(&mut self) -> Result<OpcodeResult, VMError> {
-        let current_call_frame = &mut self.current_call_frame;
-        current_call_frame.increase_consumed_gas(gas_cost::JUMPDEST)?;
+        let call_frame = &mut self.current_call_frame;
+
+        call_frame.increase_consumed_gas(gas_cost::JUMPDEST)?;
```

Comment:
While we're at it, can't we just do all in one line?

```rust
self.current_call_frame.increase_consumed_gas(gas_cost::JUMPDEST)
```

Reply:
fixed